src/main/java/bit/javaoop/FixedEmployeeNumberManager.java

```diff
@@ -0,0 +1,28 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class FixedEmployeeNumberManager extends AbstractManager {
+
+    private int hiringAbility;
+
+    public FixedEmployeeNumberManager(String name, double salary, ArrayList<AbstractEmployee> employees_list, int hiringAbility) {
+        super(name, salary, employees_list);
+        this.hiringAbility = hiringAbility;
```

Comment:
I think something like `employeeHiringLimit` or  `employeeHiringMaxCount`  would be a better name :)